src/app-layout/visual-refresh-toolbar/notifications/styles.scss

```diff
@@ -2,7 +2,13 @@
  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  SPDX-License-Identifier: Apache-2.0
 */
+@use '../../../internal/styles/tokens' as awsui;
+
 .sticky-notifications {
   position: sticky;
   z-index: 850;
 }
+
+.has-notifications-content {
+  padding-block-start: awsui.$space-scaled-xs;
```

Comment:
I'm seeing the padding still persist in https://d21d5uik3ws71m.cloudfront.net/components/bb292b4cc55f3ea99461ce066e41c6e117158b2c/dev-pages/index.html#/light/app-layout/with-table?appLayoutToolbar=true

```
.awsui_has-notifications-content_154bi_1h0eg_14:not(#\9) {
    padding-block-start: var(--space-scaled-xs-t3am3w, 8px);
}
```

Reply:
fixed